indico/modules/events/management/program_codes.py

```diff
@@ -49,13 +49,22 @@ def _make_date_placeholder(class_name, name, render_kwarg, date_format, descript
     return type(str(class_name), (DatePlaceholder,), dct)
 
 
-class ContributionIDPlaceholder(Placeholder):
+class ContributionIDPlaceholder(ParametrizedPlaceholder):
     name = 'id'
-    description = _('The ID of the contribution')
+    description = _('The ID of the contribution, optionally padded with zeros to a specified length '
```

Comment:
> The ID of the contribution (optionally padded with zeros)

might be enough, since the param is documented in the param list. you could set `param_friendly_name = 'length'` as well to have something nicer than "param" in the help text there - but feel free to keep an example in there as well!